src/bin/install.rs

```diff
@@ -123,14 +123,20 @@ pub fn execute(options: Options, config: &Config) -> CliResult<Option<()>> {
         try!(SourceId::for_central(config))
     };
 
-    let krate = options.arg_crate.as_ref().map(|s| &s[..]);
+    let krate = options.arg_crate.as_ref().map(|s| s.iter().map(|t| &t[..]).collect::<Vec<_>>());
```

Comment:
The other versions of this statement are actually just converting `Option<&String>` to `Option<&str>` but I don't think that's necessary if we have `Vec<String>`, so this part can probably just get removed.